tests/snap/code_completion/local_shadow.cpp

```
@@ -0,0 +1,14 @@
+///
+/// ## Local shadowing a global — both the local and the global name it shadows complete
+///
+/// - status: supported
+/// - order: 12
+
+// error-ok: the completion prefix dangles as an unfinished statement.
+int counter_global;
+
+void bar() {
+    int counter_local;
+    int v = coun§(pos);
```

Cppcheck (2.21.0)

[error] The code contains unhandled character(s) (character code=194). Neither unicode nor extended ascii is supported.

(unhandledChar)